src/Microsoft.DocAsCode.Plugins/BuildPhase.cs

```diff
@@ -5,9 +5,7 @@ namespace Microsoft.DocAsCode.Plugins
 {
     public enum BuildPhase
     {
-        PreBuild,
-        Build,
+        PreBuildBuild,
```

Comment:
we plan to split original build process to different phases.
1) prebuild, build
2)postbuild, save, apply templates.
and each phase could be incremental or not

---
In reply to: [89570051](https://github.com/dotnet/docfx/pull/923#discussion_r89570051) [](ancestors = 89570051)